src/instrumenter/instrumentation_context.ts

```diff
@@ -375,10 +377,12 @@ export class InstrumentationContext {
 
     private readonly litAdjustMap = new Map<Literal, ASTNode>();
 
+    private readonly unitsNeedsImports = new Map<SourceUnit, string>();
```

Comment:
In the future we may use this to say a single SourceUnit needs multiple different imports. Currently the type of this map allows only 1 import path per unit. Could you change this to:
```
    private readonly unitsNeedsImports = new Map<SourceUnit, Set<string>>();
```

Reply:
Addressed in 10c64ec